packages/core/src/types.ts

```diff
@@ -190,10 +190,21 @@ export const PropertySortOrder = {
 export type PropertySortOrder =
   (typeof PropertySortOrder)[keyof typeof PropertySortOrder];
 
+export const NamingConvention = {
+  CAMEL_CASE: 'camelCase',
+  PASCAL_CASE: 'PascalCase',
+  SNAKE_CASE: 'snake_case',
+  KEBAB_CASE: 'kebab-case',
+} as const;
+
+export type NamingConvention =
+  (typeof NamingConvention)[keyof typeof NamingConvention];
+
 export type OutputOptions = {
   workspace?: string;
   target?: string;
   schemas?: string;
+  namingConvention: NamingConvention;
```

Comment:
`OutputOptions` are values ​​specified by the user in the configuration file. Therefore, please consider `undefined` as there is a possibility that it is not input.
`NormalizedOutputOptions` is the value used by the system calculated based on the user's input. The default is entered here, so there is no `undefined`.

Reply:
Oh I see